plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -90,9 +92,25 @@ global proc mayaUSDRegisterStrings() {
     register("kSaveOptionNoPromptAnn", "You can re-enable this prompt under File|Save Scene Options.");
     register("kSaveOptionTitle", "Save USD Options");
     register("kSaveOptionUnsavedEdits", "You have unsaved USD edits. How would you like to proceed?");
+    register("kUSDSelectionMode", "USD Selection Mode");
+    register("kUSDSelectionModeAnn", "Assign selection categories to prims via the Attribute Editor to activate selection modes in this menu. Choose a selection mode to reflect changes when you select prims in the Viewport and Outliner. Note: By default, the fallback selection mode is by prim. If a selection mode is applicable, it is applied to your scene but otherwise defaults to selecting by prim.");
```

Comment:
In the JIRA item I see that the text "Attribute Editor" in this line is supposed to be bolded. You can do that with <b>text to bold</b>. See kSaveOption2GBWarning below.

Reply:
I also think you missed the second paragraph of this text. You can add the break with <br>

Reply:
Those texts are annotations on menuItem, so they will be display inside maya status bar which does not support bold text....

Reply:
I spoke with Natalia and built-in kind will not start with capital letter. So I will remove the localization.

Reply:
Oops sorry I missed that. This one is very long, does it all fit on the status line?

Reply:
It's depend of the maya UI layout you have. With my current layout yes. But the other part of the string that I did not add will not be visible. This is why I did not add it.

Reply:
Maybe just mention to Anisha where these strings will appear. I bet she though they were tooltips, and thus the length. She might reword them knowing they are on the statusbar. Also tell her, bold is not supported.

Reply:
Help text updated